AirportTicketBookingSystem/Program.cs

```diff
@@ -1,17 +1,47 @@
-﻿using System;
-using System.Collections.Generic;
+﻿using AirportTicketBookingSystem.Airport_Repository;
 using AirportTicketBookingSystem.Models;
+using AirportTicketBookingSystem.Utilties;
+using AirportTicketBookingSystem.Utilties.ManagerOptionsHandling.BookingHandling;
+using AirportTicketBookingSystem.Utilties.ManagerOptionsHandling.ErrorHandling;
+using AirportTicketBookingSystem.Utilties.ManagerOptionsHandling.FileHandling;
+using AirportTicketBookingSystem.Utilties.PassengerOptionsHandling;
+using AirportTicketBookingSystem.Utilties.PassengerOptionsHandling.AccountHandling;
+using AirportTicketBookingSystem.Utilties.PassengerOptionsHandling.BookingHandling;
+using AirportTicketBookingSystem.Utilties.PassengerOptionsHandling.FlightsHandling;
+using Microsoft.Extensions.DependencyInjection;
 
-namespace AirportTicketBookingSystem
+class Program
 {
-    class Program
+    static void Main(string[] args)
     {
-        static void Main(string[] args)
-        {
-            _ = Utilties.Utilities.GenerateFlightsAsync();
-            Utilties.Utilities.PrintMenu();
-        }
-
-       
+        var serviceProvider = new ServiceCollection()
+            .AddScoped<IFlightExport, FlightExport>()
+            .AddScoped<IFlightImport, FlightImport>()
+            .AddScoped<IFlightValidator, FlightValidator>()
+            .AddScoped<IBooking, Booking>()
+            .AddScoped<IPassenger, Passenger>()
+            .AddScoped<IFlight, Flight>()
+            .AddScoped<IPassenger, Passenger>()
```

Comment:
models is not service so no need for registering them in DI , then no need for interface for models